setup.cfg

```diff
@@ -99,6 +100,7 @@ stac = stackstac[viz]@git+https://github.com/jgrss/stackstac.git
     planetary-computer
     Rich
     wget
+    wrapt-timeout-decorator
```

Comment:
Could you actually remove this line in this PR? That package dependency was removed in the last PR that I merged.